app/serializers/rest/instance_serializer.rb

```diff
@@ -81,8 +81,15 @@ def configuration
 
   def registrations
     {
-      enabled: Setting.registrations_mode != 'none' && !Rails.configuration.x.single_user_mode,
+      enabled: registrations_enabled?,
       approval_required: Setting.registrations_mode == 'approved',
+      closed_registrations_message: registrations_enabled? ? nil : Setting.closed_registrations_message,
```

Comment:
I would prefer `reason`. 3 word property is clunky.

Reply:
My issue with `reason` is that it's unclear it's the reason the approvals are closed.